api-server/src/server/utils/donation.js

```diff
@@ -173,3 +173,9 @@ export async function handleStripeCardUpdateSession(req, app, stripe) {
   });
   return { sessionId: session.id };
 }
+
+export function inLastFiveMinutes(unixTimestamp) {
+  const currentTimestamp = Math.floor(Date.now() / 1000);
+  const timeDifference = Math.abs(currentTimestamp - unixTimestamp);
```

Comment:
```suggestion
  const timeDifference = currentTimestamp - unixTimestamp;
```
If `unixTimestamp` is always in the past, `abs` isn't necessary. If it can be in the future, something very strange is going on!